dll/cpl/desk/background.c

```diff
@@ -1012,7 +1012,7 @@ SetWallpaper(PBACKGROUND_DATA pData)
         return;
     }
 
-    if (FAILED(StringCbCat(szWallpaper, MAX_PATH, TEXT("\\Wallpaper1.bmp"))))
+    if (FAILED(StringCbCat(szWallpaper, sizeof(szWallpaper), TEXT("\\Wallpaper1.bmp"))))
```

Comment:
May be worth adding the 'W' in the function name too, since I see other String***() functions with such suffix.

Reply:
Not here: `TCHAR`...

Reply:
oh yes indeed.